speech/api/speech_gcs.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python
+# Copyright (C) 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#            http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Sample that transcribes a FLAC audio file stored in Google Cloud Storage,
+using GRPC."""
+
+import argparse
+
+from gcloud.credentials import get_credentials
+from google.cloud.speech.v1.cloud_speech_pb2 import *  # noqa
```

Comment:
Yeah - I don't usually, but the auto-generated protobuf libs are pretty opaque about what exactly they declared -_-;

Actually, I realize that I can just import the module. I'll do that... LMKWYT